lib/styleguide-view.js

```diff
@@ -0,0 +1,1267 @@
+/** @babel */
+/** @jsx etch.dom */
+
+import etch from 'etch'
+import dedent from 'dedent'
+import {html as beautifyHtml} from 'js-beautify'
+import CodeBlock from './code-block'
+import StyleguideSection from './styleguide-section'
+import ExampleSelectListView from './example-select-list-view'
+
+export default class StyleguideView {
+  constructor (props) {
+    this.uri = props.uri
+    this.collapsedSections = props.collapsedSections ? new Set(props.collapsedSections) : new Set()
+    this.sections = []
+    etch.initialize(this)
+    for (const section of this.sections) {
+      if (this.collapsedSections.has(section.name)) {
+        section.collapse()
+      } else {
+        section.expand()
+      }
+    }
+  }
+
+  destroy () {
+    this.sections = null
+  }
+
+  serialize () {
+    return {
+      deserializer: this.constructor.name,
+      collapsedSections: this.sections.filter((s) => s.collapsed).map((s) => s.name),
+      uri: this.uri
+    }
+  }
+
+  update () {
+    // intentionally empty.
+  }
+
+  getURI () {
+    return this.uri
+  }
+
+  getTitle () {
+    return 'Styleguide'
+  }
+
+  expandAll () {
+    for (const section of this.sections) {
+      section.expand()
+    }
+  }
+
+  collapseAll () {
+    for (const section of this.sections) {
+      section.collapse()
+    }
+  }
+
+  render () {
+    return (
+      <div className='styleguide pane-item native-key-bindings' tabIndex='-1'>
+        <header className='styleguide-header'>
+          <h1>Styleguide</h1>
+          <p>This exercises all UI components and acts as a styleguide.</p>
+
+          <div className='styleguide-controls btn-group'>
+            <button className='btn' onclick={() => this.collapseAll()}>Collapse All</button>
+            <button className='btn' onclick={() => this.expandAll()}>Expand All</button>
+          </div>
+        </header>
+
+        <main className='styleguide-sections'>
+          <StyleguideSection onDidInitialize={this.didInitializeSection.bind(this)} name='variables' title='Variables'>
+            <p>Use these UI variables in your package's stylesheets. They are set by UI themes and therefore your package will match the overall look. Make sure to @import 'ui-variables' in your stylesheets to use these variables.</p>
```

Comment:
Yeah.  It looks like you're using JSX (or something similar?) here, which language-javascript doesn't support as it isn't native JS.  To get it to render correctly on GitHub I think you'll need to switch the extension to `.jsx`.